kpatch-build/kpatch-elf.c

```diff
@@ -332,7 +335,27 @@ static void kpatch_find_func_profiling_calls(struct kpatch_elf *kelf)
 				break;
 			}
 		}
-#else
+#elif defined(__aarch64__)
+		if (!patchable_sec) {
+			struct section *sec = find_section_by_name(&kelf->sections,
+					      "__patchable_function_entries");
+			/*
+			 * If we can't find the __patchable_function_entries
+			 * section or there are no relocations in it then not
+			 * patchable.
+			 */
+			if (!sec || !sec->rela)
+				return;
+			patchable_sec = sec->rela;
```

Comment:
The `patchable_sec` variable is confusingly named, since it's really the relocation section for the patchable sec.  I don't think you need a separate variable for it anyway, it can just continue to be referred to by `sec->rela`.

Reply:
This was an optimisation to avoid searching for the "__patchable_function_entries" section each time. Although I'm sure it makes almost zero difference to the run time of the script.

Reply:
Oh, I see what you mean.  Maybe do the `find_section_by_name()` before the loop, and give the rela sec a more descriptive name like `patchable_rela_sec`.